Apps/Common/src/Utils/PHNValidator.cs

```diff
@@ -0,0 +1,61 @@
+﻿//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Common.Utils
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Text.RegularExpressions;
+
+    /// <summary>
+    /// Validates a Personal Health Number.
+    /// </summary>
+    public static class PHNValidator
+    {
+        private static readonly int[] PHNsigDigits = { 2, 4, 8, 5, 10, 9, 7, 3 };
+
+        /// <summary>
+        /// Validates the supplied value is a proper Personal Health Number.
+        /// </summary>
+        /// <param name="value">The Personal Health Number to validate.</param>
+        /// <returns>True if valid.</returns>
+        public static bool ValidPHN(string? value)
```

Comment:
nit: The since is static it will be fully qualified ```PHNValidator.ValidPHN()``` which looks redundant. Instead, it could simply be ```PHNValidator.IsValid()```